starlette_admin/templates/base.html

```diff
@@ -4,6 +4,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>
+    {% if favicon_url %}
+        <link rel="icon" href="{{ favicon_url }}"/>
```

Comment:
consider setting the type as well
```suggestion
        <link rel="icon" type="image/x-icon" href="{{ favicon_url }}"/>
```

Reply:
Thanks !